src/graph.rs

```diff
@@ -2410,7 +2410,30 @@ impl ModuleGraph {
     }
   }
 
-  /// Return a vector of references to module objects in the graph. Only modules
+  /// List of all the url in the graph that are statically known to be
+  /// imported with type "text" or "bytes".
+  pub fn asset_module_urls(&self) -> IndexSet<&Url> {
```

Comment:
I'm too lazy to add a test here because this is somewhat simple and a lot of setup to test. There's one in the deno cli that should be sufficient.